src/minidump-win32.cc

```diff
@@ -57,10 +57,10 @@ void CreateWin32MiniDump(_EXCEPTION_POINTERS* pep) {
     return;
   }
 
-  HANDLE hFile = CreateFileA(temp_file, GENERIC_READ | GENERIC_WRITE, 0, NULL,
+  HANDLE hFile = CreateFile(Utf8ToWide(temp_file).c_str(), GENERIC_READ | GENERIC_WRITE, 0, NULL,
                              CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, NULL);
   if (hFile == NULL) {
-    Error("failed to create minidump: CreateFileA(%s): %s",
+    Error("failed to create minidump: CreateFileW(%s): %s",
```

Comment:
You're not calling `CreateFileW`, you're calling `CreateFile`.

Reply:
Due to `-DUNICODE` `CreateFile` is defined as `CreateFileW`.

Reply:
The error message is still wrong.

Either the function being called should be changed to `CreateFileW`, or the error message should be changed to `CreateFile`, instead of `CreateFileW`.

I'm with @MrTrillian with regard to explicitly calling the `W` version of these functions.

Reply:
I agree. I will mark this as resolved then, since the error is fine, the function call needs to change.